### PR TITLE
Fix asn1

### PR DESCRIPTION
**Pull request by Juan-M-V**
A fix for asn1 fuzzer

### CODE REVIEW

File: asn1/src/main.rs [hunk 1]
```diff
@@ -1,32 +1,39 @@
 use honggfuzz::fuzz;
-use simple_asn1::*;
-use asn1::*;
+//use simple_asn1::*;
```
Comment (daphneherlambda) [inline asn1/src/main.rs]: delete this comment as this is not in use anymore

**Review by daphneherlambda — CHANGES_REQUESTED** [review]
LGTM just a detail, delete the extra comment, and add comments on what is evey step doing to give a little context on what the fuzzer does and why

File: asn1/src/main.rs (hunk 1 above)
Comment (Juan-M-V) [reply asn1/src/main.rs]: Fixed: https://github.com/lambdaclass/fuzzing_examples/pull/17/commits/b0ff795880768b03874cd94706bc19527442a6a3

File: asn1/src/main.rs [hunk 2]
```diff
@@ -1,32 +1,58 @@
 use honggfuzz::fuzz;
-use simple_asn1::*;
-use asn1::*;
+
+// This fuzzer tests the following DER strcuture:
```
Comment (daphneherlambda) [inline asn1/src/main.rs]:
```suggestion
// This fuzzer tests the following DER structure:
```
fix the typo and we´re done!

**Review by daphneherlambda — CHANGES_REQUESTED** [review]

**Review by daphneherlambda — APPROVED** [review]

**Review by MegaRedHand — APPROVED** [review]

**Pull request merged**